### PR TITLE
formatted unit.py doc

### LINKED ISSUE #3580: Fix typos in the docs - multiple PRs welcome!
**From https://github.com/MDAnalysis/UserGuide/issues/194 -- @micaela-matta**

This issue is designed to be incrementally achieved through multiple PRs. 

There are a few typos in the user guide, this issue will collect as many as we can find starting with the short list below.
Feel free to fix any typos/formatting errors we haven't noticed and add them in the comments!

- units and conversions [page]( https://docs.mdanalysis.org/2.1.0-dev0/documentation_pages/units.html?highlight=unit%20conversion#conversions), note box: "in the future me might" 
- units and conversions [page]( https://docs.mdanalysis.org/2.1.0-dev0/documentation_pages/units.html?highlight=unit%20conversion#conversions), above the note box: fix LaTeX formatting for `rho/rho0 = n/(N_A * M**-1) / rho0 where [n] = 1/Volume, [rho] = mass/Volume`
- units and conversions [page]( https://docs.mdanalysis.org/2.1.0-dev0/documentation_pages/units.html?highlight=unit%20conversion#conversions), `MDAnalysis.units.water`: fix scientific notation in`and molar mass 18.016 g mol**-1`
- density analysis [page](https://docs.mdanalysis.org/2.1.0-dev0/documentation_pages/analysis/density.html), units and conversions [page]https://docs.mdanalysis.org/2.1.0-dev0/documentation_pages/units.html?highlight=unit%20conversion#conversions) (and probably many other pages): the symbol `Å`  appears many times in different formats (`ångströms`, `Å**-3`, `Angstroms` etc). Depending on the context in the page, keep LaTeX equation style or replace with the standard symbol `Å` and correct scientific notation.

### PR DESCRIPTION
**Pull request by robotjellyzone**
Fixes #3580  - changes done for [unit and conversion page](https://docs.mdanalysis.org/2.1.0-dev0/documentation_pages/units.html?highlight=unit%20conversion#conversions)

Changes made in this Pull Request:
 - Fixed some typos
 - Fixed LaTeX formatting for `rho/rho0 = n/(N_A * M**-1) / rho0 where [n] = 1/Volume, [rho] = mass/Volume`

PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/units.py [hunk 1]
```diff
@@ -104,13 +104,14 @@
 
      factor = 1 A**-3 / (N_Avogadro * (10**-9 dm)**-3)
 
-  relative to a density rho0 in g/cm^3::
+  relative to a density rho0 in g/cm^{3}::
```
Comment (PicoCentauri) [inline package/MDAnalysis/units.py]: for a propper rendering of the exponent maybe try :_math:\`g/cm^3\`_

File: package/MDAnalysis/units.py [hunk 2]
```diff
@@ -104,13 +104,14 @@
...
 
     M(H2O) = 18 g/mol   Molar mass of water
 
     factor = 1/(1e-24 * N_Avogadro / M(H2O))
+  ::
 
-  from `rho/rho0 = n/(N_A * M**-1) / rho0`  where `[n] = 1/Volume`, `[rho] = mass/Volume`
+    from rho/rho0 = n/(N_A * M**-1) / rho0   where [n] = 1/Volume, [rho] = mass/Volume
```
Comment (PicoCentauri) [inline package/MDAnalysis/units.py]: Same as above. Try using the :math: syntax and not a code block.

**Review by PicoCentauri — CHANGES_REQUESTED** [review]
Good start. To have a even clearer format I suggest to use the :math: syntax.

File: package/MDAnalysis/units.py (hunk 1 above)
Comment (robotjellyzone) [reply package/MDAnalysis/units.py]: thank you so much for your review. Will do the changes. its first time I am using rst files and doing changes in its related documentation (as till now i just have used only markdown 😅 )

File: package/MDAnalysis/units.py [hunk 3]
```diff
@@ -104,13 +104,15 @@
...
      factor = 1 A**-3 / (N_Avogadro * (10**-9 dm)**-3)
 
-  relative to a density rho0 in g/cm^3::
+  relative to a density rho0 in :math:`g/cm^3`::
 
     M(H2O) = 18 g/mol   Molar mass of water
 
     factor = 1/(1e-24 * N_Avogadro / M(H2O))
+  
+  from :math:`rho/rho0 = n/(N_A * M^-1) / rho0`
```
Comment (PicoCentauri) [inline package/MDAnalysis/units.py]: You could also type set the greek symbols here 
```suggestion
  from :math:`\rho/\rho_0 = n/(N_A * M^{-1}) / \rho_0`
```

File: package/MDAnalysis/units.py [hunk 4]
```diff
@@ -104,13 +104,15 @@
...
 
-  from `rho/rho0 = n/(N_A * M**-1) / rho0`  where `[n] = 1/Volume`, `[rho] = mass/Volume`
+  where :math:`[n] = 1/Volume`, :math:`[rho] = mass/Volume`
```
Comment (PicoCentauri) [inline package/MDAnalysis/units.py]: same here 

```suggestion
  where :math:`[n] = 1/Volume`, :math:`[\rho] = mass/Volume`
```

**Review by PicoCentauri — CHANGES_REQUESTED** [review]
Already much better! There is also one PEP8 error as @pep8speaks reminds us.

Would you also mind changing the typo in the note box?

In the future **m**e might move - > In the future **w**e might move

File: package/MDAnalysis/units.py (hunk 3 above)
Comment (robotjellyzone) [reply package/MDAnalysis/units.py]: ok i will do the changes. one thing, can you tell which pep8 error you are talking about ? @PicoCentauri

**Comment by PicoCentauri** [comment]
[See above](https://github.com/MDAnalysis/mdanalysis/pull/3634#issuecomment-1097726233)

File: package/MDAnalysis/units.py (hunk 3 above)
Comment (PicoCentauri) [reply package/MDAnalysis/units.py]: [See above](https://github.com/MDAnalysis/mdanalysis/pull/3634#issuecomment-1097726233)

**Comment by robotjellyzone** [comment]
> [See above](https://github.com/MDAnalysis/mdanalysis/pull/3634#issuecomment-1097726233)

ok so i need to remove blank line ?

**Comment by PicoCentauri** [comment]
Looks good. Maybe @micaela-matta wants to take a look? For less clicking: [Current version of the units page ](https://mdanalysis--3634.org.readthedocs.build/en/3634/documentation_pages/units.html)

**Comment by robotjellyzone** [comment]
@lilyminium can you also take a look at my pr ! thanks !

**Review by PicoCentauri — APPROVED** [review]
LGTM, but lets wait for another couple of eyes. I am not the expert for the documentation.

**Comment by lilyminium** [comment]
@robotjellyzone is your name actually Robot Jelly? I thought you had introduced yourself as Kavya Bisht. If the name is incorrectly listed in AUTHORS, could you please make a new PR to fix that?

**Comment by robotjellyzone** [comment]
> @robotjellyzone is your name actually Robot Jelly? I thought you had introduced yourself as Kavya Bisht. If the name is incorrectly listed in AUTHORS, could you please make a new PR to fix that?

In the authors, i thought we need to write the name of my github but my actual name is Kavya Bisht . so shall i just add  this name in new pr? but anyways i will make one pr for another issue then i will change at that time if its anyway required @lilyminium

**Comment by lilyminium** [comment]
> In the authors, i thought we need to write the name of my github but my actual name is Kavya Bisht . so shall i just add this name in new pr?

Yes please -- we ask for the username in the CHANGELOG, but people's (preferred) names in the AUTHORS list :)

**Comment by robotjellyzone** [comment]
> > In the authors, i thought we need to write the name of my github but my actual name is Kavya Bisht . so shall i just add this name in new pr?
> 
> Yes please -- we ask for the username in the CHANGELOG, but people's (preferred) names in the AUTHORS list :)

dont worry . i will make new pr for another issue ie second issue (as this was just my first pr) at that time i will change:)

**Pull request merged**